src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/gtfs/ScheduleTestsIT.java

```diff
@@ -0,0 +1,132 @@
+  
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    createPtvSettings();
+    
+    getPtvMocker()
+      .mockOrganizations("0de268cf-1ea1-4719-8a6e-1150933b6b9e")
+      .startMock();
+
+    waitApiListCount("/organizations", 1);
+    
+    createGtfsSettings(getOrganizationId(0));
+
+    waitApiListCount(String.format("/organizations/%s/transportSchedules", getOrganizationId(0)), 17); 
+  }
+
+  @After
+  public void afterClass() {
+    String organizationId = getOrganizationId(0);
+    getPtvMocker().endMock();
+    deletePtvSettings();
+    deleteGtfsSettings(organizationId);
+  }
+  
+  @Test
+  public void testListSchedules() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportSchedules", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(17))
+      .body("id[1]", notNullValue())
+      .body("name[1]", is("L+ Mikkeli"))
+      .body("days[1].size()", is(1))
+      .body("days[1][0]", is(6))
+      .body("exceptions[1].size()", is(10))
+      .body("exceptions[1][1].type", is("REMOVE"))
+      .body("exceptions[1][1].date",sameInstant(getInstant(2016, 7, 30, 0, 0, TIMEZONE_ID)))
+      .body("startDate[1]", sameInstant(getInstant(2016, 5, 1, 0, 0, TIMEZONE_ID)))
+      .body("endDate[1]", sameInstant(getInstant(2024, 4, 30, 0, 0, TIMEZONE_ID)));
+  }
+  
+  @Test
+  public void testFindSchedule() {
+    String organizationId = getOrganizationId(0);
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/transportSchedules/{scheduleId}", organizationId, getOrganizationScheduleId(organizationId, 10))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id", notNullValue())
+      .body("name", is("Ma+++"))
+      .body("days.size()", is(1))
+      .body("days[0]", is(1))
+      .body("exceptions.size()", is(22))
+      .body("exceptions[1].type", is("REMOVE"))
+      .body("exceptions[1].date",sameInstant(getInstant(2017, 1, 23, 0, 0, TIMEZONE_ID)))
+      .body("startDate", sameInstant(getInstant(2016, 5, 1, 0, 0, TIMEZONE_ID)))
+      .body("endDate", sameInstant(getInstant(2024, 4, 30, 0, 0, TIMEZONE_ID)));
+  }
+  
+  @Test
+  public void testOrganizationSchedulesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationScheduleId = getOrganizationScheduleId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/transportSchedules/%s", organizationId, organizationScheduleId));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "/organizations/%s/transportSchedules/%s" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)